pwiz_tools/Skyline/Model/AbstractModificationMatcher.cs

```diff
@@ -580,16 +580,15 @@ private string NoteIfAnnotationMzDisagrees(LibKey key, SpectrumPeaksInfo.MI peak
                 var averageMass = charge.MassFromMz(peak.Mz, MassType.Average);
 
                 if (!(peakAnnotation.Ion.MonoisotopicMass.Equals(monoisotopicMass,
-                          Settings.TransitionSettings.Instrument.MzMatchTolerance) ||
-                      peakAnnotation.Ion.AverageMass.Equals(averageMass,
-                          Settings.TransitionSettings.Instrument.MzMatchTolerance)))
+                          Settings.TransitionSettings.Instrument.IonMatchMzTolerance) ||
+                      Settings.TransitionSettings.Instrument.IonMatchMzTolerance.IsWithinTolerance(peakAnnotation.Ion.AverageMass, averageMass)))

```

Comment:
It's confusing that the code for the mono mass uses the ".Equals(double, MzMatchTolerance)" function, but the average mass uses "IsWithinTolerance".
Could this expression be rewritten so that the same function is used for comparing mono mass and average mass?

Reply:
Agree.